### PR TITLE
app-misc/skim: bump to 0.9.4

### PR DESCRIPTION
**Pull request by dm9pZCAq**
changes:
* version bumped
* added `*-completion` use-flags
* added https://crates.io/crates/skim `HOMEPAGE` 
* changed `DESCRIPTION` from `Command-line fuzzy finder` to `Fuzzy Finder in rust!` as on [skim git](https://github.com/lotabout/skim) (should it be changed in `0.9.3` as well?)

Package-Manager: Portage-3.0.18, Repoman-3.0.3
Signed-off-by: dm9pZCAq <v@0x0c.xyz>

### CODE REVIEW

File: app-misc/skim/skim-0.9.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+		doins "plugin/${PN}.vim"
+	}
+
+	# install bash, zsh, fish completion and keybindings
+	# since provided completions override a lot of commands,
+	# install to /usr/share
+	insinto "/usr/share/${PN}"
+	use bash-completion && doins shell/*.bash
+	use zsh-completion  && doins shell/*.zsh
+	use fish-completion && doins shell/*.fish
```
Comment (juippis) [inline app-misc/skim/skim-0.9.4.ebuild:166]: We always install these files and don't control them via USE flags, 
https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0301

File: app-misc/skim/skim-0.9.4.ebuild [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+"
+SRC_URI="$(cargo_crate_uris ${CRATES})"
+
+LICENSE="Apache-2.0 BSD BSD-2 CC0-1.0 MIT MPL-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+IUSE="tmux vim bash-completion zsh-completion fish-completion"
+
+RDEPEND="
+	tmux? ( app-misc/tmux )
```
Comment (juippis) [inline app-misc/skim/skim-0.9.4.ebuild:139]: Put this as `optfeature` instead, and always install the binary.
https://devmanual.gentoo.org/eclass-reference/optfeature.eclass/index.html

File: app-misc/skim/skim-0.9.4.ebuild [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+inherit cargo
+
+DESCRIPTION="Fuzzy Finder in rust!"
+HOMEPAGE="
+	https://crates.io/crates/skim
+	https://github.com/lotabout/skim
+"
+SRC_URI="$(cargo_crate_uris ${CRATES})"
+
+LICENSE="Apache-2.0 BSD BSD-2 CC0-1.0 MIT MPL-2.0"
```
Comment (juippis) [inline app-misc/skim/skim-0.9.4.ebuild:133]: Have you checked the latest state with *cargo-license*?

**Review by juippis — COMMENTED** [review]
And again, your commit is missing the mandatory proper sign-off, 
https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin

File: app-misc/skim/skim-0.9.4.ebuild (hunk 3 above)
Comment (dm9pZCAq) [reply app-misc/skim/skim-0.9.4.ebuild:133]: yes

**Comment by AndrewAmmerlaan** [comment]
Closing as obsolete, latest version in ::gentoo is 0.10.4

**Pull request closed without merging**